point-of-sale-app/pom.xml

```diff
@@ -0,0 +1,60 @@
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
No Gradle ;) The pom xml files get bloated very quickly, build performance, etc.

Reply:
Are you suggesting we consider moving to gradle from maven?

Reply:
It's okay for the app, but I was wondering if you had considered. It's a better system though.

Reply:
I have used it for pretty basic projects. Interesting, that would have been good. 
I didn't consider it. Mainly because of the timeline I went with everything that I was already familiar with!